src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationTest.java

```diff
@@ -30,8 +30,8 @@ private void init() {
     new Navigate(driver).toPage(MercurySubpages.MAIN_PAGE);
   }
 
-  @Test(groups = "navigation-openAndCloseNavigationAndItsSubmenu")
-  public void openAndCloseNavigationAndItsSubmenu() {
+  @Test(groups = "mercury_navigation_openAndCloseNavigationAndItsSubMenu")
+  public void mercury_navigation_openAndCloseNavigationAndItsSubMenu() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)